grafana/dashboards/health.json

```diff
@@ -1260,6 +1260,118 @@
         "alignLevel": null
       }
     },
+    {
```

Comment:
I'm assuming most of the changes in this file are autogenerated; if so I'm curious how? Is there provided utility in updating what the `health.json` should look like?

Reply:
Half autogenerated, half hand-edited. My workflow for Grafana changes usually looks like:
1) Edit dashboard in the Grafana UI
2) Export dashboard JSON
3) Overwrite existing dashboard.json file in git repo
4) Hand-edit changes to clean up any extraneous unwanted changes